### PR TITLE
Separate production settings

### PR DESCRIPTION
**Pull request by prateekkumarweb**
Separate the production settings file. This file has to be edited during deployment. It can be merged to master now.

### CODE REVIEW

**Comment by prateekkumarweb** [comment]
@vishwakftw Can you configure the linter to ignore the `settings_production.py` file. After that, you can merge this PR.

File: autojudge/settings_production.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# To use these settings during development, run with
+# python manage.py runserver --settings=autojudge.settings_production
+
+import os
+from typing import List
+from .settings import *
```
Comment (vishwakftw) [inline autojudge/settings_production.py]: Can you add a comment like this below?

`# noqa: F403`

You can look at settings.py or submission_watcher_saver.py for reference.

**Comment by prateekkumarweb** [comment]
> Shall I merge?

Yes

**Comment by vishwakftw** [comment]
GO FOR AUTOSEQUENCE START.

**Pull request merged**